src/controllers/payment.js

```diff
@@ -1,3 +1,28 @@
-// import { deleteReviewById } from '../queries/deleteReviewById';
-//import Business from '../queries/Business';
+import Payment from '../queries/Payment';
 
+const stripe = require('stripe')(process.env.STRIPE_SECRET_KEY);
+
+export async function handleCharge (req, res) {
+  const token = req.body.paymentData.token;
```

Comment:
is this a timed token ? if not, i think we should save it in the db for future purchases